<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubsonicSettingsPage</class>
 <widget class="QWidget" name="SubsonicSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subsonic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="LoginStateWidget" name="login_state" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="server_group">
     <property name="title">
      <string>Server details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="server_label">
        <property name="text">
         <string>Server URL</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="username_label">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="username"/>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="server"/>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="usesslv3">
        <property name="text">
         <string>Use SSLv3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="verifycert">
        <property name="text">
         <string>Verify Server certificate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="login">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginStateWidget</class>
   <extends>QWidget</extends>
   <header>widgets/loginstatewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>server</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>usesslv3</tabstop>
  <tabstop>login</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
